bitnami/harbor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: harbor
-version: 5.0.10
```

Comment:
As it is adding a new feature, could you bump the minor version instead?

Reply:
Done ;)